src/ServiceProvider.php

```diff
@@ -131,6 +131,8 @@ protected function configureHealth()
     {
         HealthRepository::customApplicationData(fn () => [
             'butlerService' => ltrim(InstalledVersions::getPrettyVersion('glesys/butler-service'), 'v'),
+            'laravelOctane' => ltrim(InstalledVersions::getPrettyVersion('laravel/octane'), 'v'),
```

Comment:
I think it throws an exception if the package is not installed? https://getcomposer.org/doc/07-runtime.md#knowing-the-version-of-package-x

We might need to check if it installed before checking the version?

Reply:
It is always installed: https://github.com/glesys/butler-service/blob/master/composer.json#L16

Reply:
Oh. TIL. :+1: